nameresolution/mdns/mdns.go

```diff
@@ -384,81 +673,89 @@ func (m *resolver) refreshAllApps(ctx context.Context) error {
 }
 
 // browse will perform a non-blocking mdns network browse for the provided app id.
-func (m *resolver) browse(ctx context.Context, appID string, onEach func(ip string)) error {
+func (m *Resolver) browse(ctx context.Context, appID string, onEach func(ip string)) error {
```

Comment:
quick question why is the callback named `onEach`? The only non nil value it gets is an `onFirst` method?

Reply:
legacy reasons, this was originally a non nil value in refreshApp but is no longer used as browse updates the cache directly.